app/views/bloghome.php

```diff
@@ -1,38 +1,42 @@
 <?php
+use Prismic\LinkResolver;
+use Prismic\Dom\RichText;
+use Prismic\Dom\Date;
 
 $prismic = $WPGLOBAL['prismic'];
 $bloghome = $WPGLOBAL['bloghome'];
 $posts = $WPGLOBAL['posts'];
 
-$title = $bloghome->getText("blog_home.headline");
+$title = RichText::asText($bloghome->data->headline);
 $isBloghome = true;
 
-$imageUrl = $bloghome->getImage('blog_home.image') ? $bloghome->getImage('blog_home.image')->getUrl() : '';
+$imageUrl = $bloghome->data->image->url;
+$linkResolver = $prismic->linkResolver;
 
 ?>
 
 <?php include 'header.php'; ?>
 
 <div class="home">
   <div class='blog-avatar' style='background-image: url("<?= $imageUrl ?>");'></div>
-  <h1 class='blog-title'><?= $title ?></h1>
-  <p class='blog-description'><?= $bloghome->getText('blog_home.description') ?></p>
+  <h1 class='blog-title'><?= RichText::asHtml($bloghome->data->headline) ?></h1>
```

Comment:
You already have the headline title defined above in the $title variable. No need to output this as HTML here.